.ci/helper_shellchecks.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Based on https://github.com/koreader/koreader/blob/master/.ci/helper_shellchecks.sh
+
+# shellcheck disable=2016
+mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' && git submodule --quiet foreach '[ "$path" = "base" -o "$path" = "platform/android/luajit-launcher" ] || git grep -lE "^#!(/usr)?/bin/(env )?(bash|sh)" | sed "s|^|$path/|"' && git ls-files ./*.sh; } | sort | uniq)
```

Comment:
Consider splitting this long line

Reply:
Better yet, I didn't read through the script in detail since it was a bit of a quick breakfast commit and the part about submodules is irrelevant in this repo.